cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,54 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
-
+  beforeEach(() => {
+    cy.visit('/webtables');
+  });
+  it('should display the total number of pages when new rows are added', () => {
+    cy.createWorkers(4, 11);
+    cy.get('.-totalPages').should('contain.text', '2');
+  });
+  it('should allow rows count selection', () => {
+    cy.createWorkers(4, 11);
+    cy.get('select').select('20 rows');
+    cy.get('.rt-td').should('contain.text', 'worker№' + 11);
+  });
+  it('should allow to add a new worker record', () => {
+    cy.createWorkers(4, 4);
+    cy.get('.rt-td').should('contain.text', 'worker№' + 4);
+  });
+  it('should allow to delete a worker record', () => {
+    cy.createWorkers(4, 4);
+    cy.get('#delete-record-4').click();
+    cy.contains('worker№' + 4).should('not.exist');
+  });
+  it('should allow to delete all worker records', () => {
+    cy.createWorkers(4, 10);
```

Comment:
why do you need to add workers here? you could just remove 3 workers that already exist in the table

Reply:
you still adding them. Not a critical but it is not necessary and just slow down the test